source/install.sql

```diff
@@ -29,6 +29,7 @@ whenever oserror exit failure rollback
 prompt Switching current schema to &&ut3_owner
 prompt &&line_separator
 alter session set current_schema = &&ut3_owner;
+alter session set PLSQL_WARNINGS='ENABLE:ALL';
```

Comment:
Warnings are enabled as part of development install. They were removed from the main install on purpose so users don't get worried that something is wrong.